virttest/utils_net.py

```diff
@@ -3349,9 +3349,9 @@ def __arping(ip, macs, dev, timeout, session=None, **dargs):
 
         mac_regex = "|".join("(%s)" % mac for mac in macs)
         regex = re.compile(r"\b%s\b.*\b(%s)\b" % (ip, mac_regex), re.I)
-        arping_bin = utils_path.find_command("arping")
+        arping_bin = utils_path.find_command("arping").strip()
```

Comment:
I have read code of find_comman(), I don't think the '\n' could be brought in here. Maybe we don't need to change this line.

Reply:
Yes, I agree with you.

Reply:
But in my latest test env, it has "\n" indeed. I have pasted the output of arping_cmd in job.log above. We can tell that there is "\n" arping_cmd.
And the fix does work.
Maybe you could try on my test env?

Reply:
Sorry, I misunderstood you just now. I will restore this line.

Reply:
Thanks for your comment.😄 
Updated.